.github/workflows/dep-update-review.yml

```diff
@@ -0,0 +1,72 @@
+name: Dependency Update Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - "**Cargo.lock"
+      - "**Cargo.toml"
+
+jobs:
+  generate-report:
+    runs-on: ubuntu-latest-xl
+    name: Generate update review report
+    outputs:
+      update-report: ${{ steps.set-update-report.outputs.report }}
+      update-report-length: ${{ steps.set-update-report.outputs.length }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: checkout base ref
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+          path: diem-base
+      - name: install depdive
+        run: cargo install --git https://github.com/diem/whackadep --branch main depdive
```

Comment:
Lets make sure we're pinning to a particular revision, that way if something happens to change at the tip of main that breaks compatibility this job wont be impacted.